packages/fixie/src/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { toTextStream } from 'ai-jsx/stream';
+import { ChatCompletion, SystemMessage, UserMessage } from 'ai-jsx/core/completion';
+
+function HelloWorld({ message }: { message: string }) {
+  return (
+    <ChatCompletion>
+      <SystemMessage>Respond to the user using some variant of the phrase "Hello World!".</SystemMessage>
+      <UserMessage>{message}</UserMessage>
+    </ChatCompletion>
+  );
+}
+
+export default function handleMessage(message: string) {
+  return toTextStream(<HelloWorld message={message} />);
```

Comment:
If we're trying to make the platform hosting as seamless as possible, should we not even do `toTextStream` – instead, have this file just export the component, and the platform wraps with `toTextStream`?

Reply:
let me give that a shot